docs/comparison.md

```diff
@@ -1,42 +1,237 @@
+| STT - On device         | N/A                                   | grokotron<br>(internal plugin)        | N/A                                   |
+| STT - On premises       | N/A (?)                               | N/A                                   | N/A (?)                               |
+| STT - Cloud             | mycroft (selene)<br>(internal plugin) | mycroft (selene)<br>(internal plugin) | mycroft (selene)<br>(internal plugin) |
+| TTS - On device         | mimic<br>(internal plugin)            | mimic 3<br>(internal plugin)          | mimic<br>(internal plugin)            |
+| TTS - On premises       | N/A (?)                               | N/A (?)                               | N/A (?)                               |
+| TTS - Cloud             | mimic2<br>(internal plugin)           | N/A (?)                               | mimic2<br>(internal plugin)           |
+| VAD                     | N/A<br>(missing feature)              | silero (hardcoded)                    | N/A<br>(missing feature)              |
+
+
+## Functionality
+
+MNOs - Mycroft/Neon/OVOS
+
+### OpenVoiceOS
+
+|                                    | ovos-buildroot                      | ovos-arch                           | ovos-raspbian                       |
+|------------------------------------|-------------------------------------|-------------------------------------|-------------------------------------|
+| **Skill Frameworks**               |                                     |                                     |                                     |
+| Mycroft Audio Service<br>(bus api) | Yes                                 | Yes                                 | Yes                                 |
+| Adapt Intents                      | Yes                                 | Yes                                 | Yes                                 |
+| Padatious Intents                  | Yes                                 | Yes                                 | Yes                                 |
+| OCP Skills                         | Yes                                 | Yes                                 | Partial<br>(audio only)             |
+| MNOS Common Query Skills           | Yes                                 | Yes                                 | Yes                                 |
+| MNOs Fallback Skills               | Yes                                 | Yes                                 | Yes                                 |
+| MNOs Skills                        | Yes                                 | Yes                                 | Yes                                 |
+| Mycroft Common Play Skills         | Partial<br>(imperfect compat layer) | Partial<br>(imperfect compat layer) | Partial<br>(imperfect compat layer) |
+| **Media Frameworks**               |                                     |                                     |                                     |
```

Comment:
Most of the media frameworks here, I would like to include in the image.  Maybe add "WIP"